src/types/params/leads.ts

```diff
@@ -1,7 +1,7 @@
 import { LeadSortTypeParams, SortParams } from "../../types/sort"
 
 import { PaginationParams } from "./index"
-
 export interface LeadQueryParams extends PaginationParams {
   sort: SortParams<LeadSortTypeParams>
+  searchQuery: string
```

Comment:
Should the param be mandatory? Look like your are defaulting it in the search function...

Reply:
@lkappeler you are right, it should be optional, my bad 🤦‍♂️